lib/datasource/remote/api/eos_repo/eos_repository.dart

```diff
@@ -71,7 +73,10 @@ abstract class EosRepository {
   }
 
   ErrorResult mapEosError(dynamic error) {
-    print('mapEosError: $error');
+    if (kDebugMode) {
+      print('mapEosError: $error');
+    }
+    FirebaseCrashlytics.instance.log('mapEosError: $error');
```

Comment:
❤️

Reply:
If this removes EOS errors completely from the device log, then I don't agree with this change. 

If crashlytics prints the errors to console as well, then OK. 

EOS errors must be in the device logs, anyway.